### DISCUSSION TITLE
User list including channels

### DISCUSSION THREAD

**Discussion by herzkerl** [rusq/slackdump#167] [Q&A] [answered]
Dear community,

is there a way to export a list of all users including their channels (i.e. which channels does the user belong to)—or vice versa?

Thanks in advance!

**Comment by rusq** [comment, marked as answer]
Hey Ben, thanks for your question - currently it's not possible.

Users and channels can be exported separately, but not user->channel link (or vice versa).

There is an issue #125 that is somewhat related, I think it would make sense to include this suggestion and implement it a part of that issue.
